packages/editor/src/components/browser/CmsBrowser.tsx

```diff
@@ -64,7 +65,7 @@ const CmsBrowser = ({ value, onChange, noApiCall, typeFilter, onDoubleClick }: C
             <ExpandableCell
               cell={cell}
               title={cell.row.original.name}
-              icon={cell.row.original.type === 'FOLDER' ? IvyIcons.GoToSource : undefined}
+              icon={cell.row.original.type === 'FOLDER' ? IvyIcons.FolderOpen : IvyIcons.File}
```

Comment:
what about content object strings?

Reply:
I believe Anais has not yet designed a suitable icon for it, or is the following icon intended for strings?
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/4cd25203-b533-46d3-a86c-f05d3e2aef1f)